data/build_image.wf.json

```diff
@@ -68,6 +71,9 @@
         }
       ]
     },
+    "resize-disk":{ 
+      {{.ResizeDisks}}
+    },
```

Comment:
If OEMSize == "", will the stateful partition ever be resized up after this step?